src/docs/howto-users-and-authentication/setup-oidc-proxy/index.md

```diff
@@ -299,47 +277,48 @@ Console for this variable.
 `KEY_ID`
 
 The value will be used as the `kid` header in ID tokens that are passed
-to Sharetribe when a user logs in with LinkedIn. It is also used as the
+to Sharetribe when a user logs in with Github. It is also used as the
 `kid` attribute of the JSON Web key that the proxy serves in an
 endpoint. Even though using a _kid_ value in your keys is not
 compulsory, we heavily recommend using it with your token and the JWK.
 For example, key caching in the Sharetribe API relies heavily on it.
+This value can be a random string.
 
 ### Add Passport module dependency
 
 We are using [Passport.js](http://www.passportjs.org) library for
 handling the authentication with different identity providers like with
 Facebook and Google. The library offers multiple authentication
-strategies and there's also a strategy for Linkedin which we are going
-to use in this example.
+strategies and there's also
+[a strategy for Github](https://www.passportjs.org/packages/passport-github2/),
+which we are going to use in this example.
 
-Add the following entry to the `dependencies` map in `package.json` and
-run `yarn install`:
+Run the following command in your terminal to install the package:
 
-```js
-"passport-linkedin-oauth2": "^2.0.0",
+```shell
+yarn add passport-github2
 ```
 
-### Implement the LinkedIn login flow in Sharetribe Web Template backend
+### Implement the Github login flow in Sharetribe Web Template backend
 
 Next, let's add a new file to the template that handles authentication
-to LinkedIn. You can find the complete file here:
+to Github. You can find the complete file here:
 
-- [linkedin.js](/tutorial-assets/linkedin.js)
+- [github.js](/tutorial-assets/github.js)
 
 Place the file in `server/api/auth` folder inside the template:
 
 ```shell
 └── server
     └── api
         └── auth
-            └── linkedin.js
+            └── github.js
 ```
 
-The biggest difference between LinkedIn login and e.g. Facebook login
-which has first-class support in Sharetribe is that we need to use
+The biggest difference between Github login and e.g. Facebook login,
+which has first-class support in Sharetribe, is that we need to use
 _createIdToken_ helper function to create the id token from the
-information we fetched from LinkedIn. This new id token is then passed
+information we fetched from Github. This new id token is then passed
 forward to Sharetribe as _idpToken_ parameter.
```

Comment:
Comment: This part feels a bit detached from the rest of this section. I was initially confused because I had mentally adjusted to following a series of steps and my brain was trying to read this as the next step. It also has a code section so when scanning through this I was trying to figure out where should I be pasting this code. Maybe just adding "Note:" to the beginning of this paragraph or wrapping it in an info box?

Reply:
Good catch! I now wrapped this in an info box and removed the code snippet (because code snippets in info boxes don't really work with the current CSS).